File: repos_estudiantes_ALVAREZ_primera_entrega/GUTIERREZ_GARCIA_CARLA_ALEJANDRA/IntroduccionPOO/Ejercicio1/Persona.java
public class Persona{
    //Crea una clase Persona con nombre, edad y ciudad
    public String nombre,ciudad;
    public int edad;

    public Persona(String nombre, int edad,String ciudad){
        this.nombre = nombre;
        this.edad = edad;
        this.ciudad = ciudad;
    }
    //a) Agrega un método para mostrar el saludo: “Hola, soy {nombre} de {ciudad}”
    public void Saludo(){
        System.out.println("Hola, soy " + nombre + " de " + ciudad);
    }
    //c) Agrega un método para verificar si es mayor de edad
    public void verificarMayorEdad(){
        if(this.edad>=18){
            System.out.println("Mayor de edad");
        }
        else{
            System.out.println("Menor de edad");
        }
    }
    public static void main(String[] args) {
        //b) Crea tres personas y muestra su saludo
        Persona p1= new Persona("Carol",18,"La Paz");
        Persona p2= new Persona("Maria",22,"Cochabamba");
        Persona p3= new Persona("Juan",15,"Chuquisaca");
        p1.Saludo();
        p2.Saludo();
        p3.Saludo();
        p1.verificarMayorEdad();
        p2.verificarMayorEdad();
        p3.verificarMayorEdad();
    }
}

File: repos_estudiantes_ALVAREZ_primera_entrega/GUTIERREZ_GARCIA_CARLA_ALEJANDRA/IntroduccionPOO/Ejercicio1/Persona.py
class Persona:
    # Crea una clase Persona con nombre, edad y ciudad
    def __init__(self, nombre, edad, ciudad):
        self.nombre = nombre
        self.edad = edad
        self.ciudad = ciudad

    # a) Agrega un método para mostrar el saludo
    def saludo(self):
        print(f"Hola, soy {self.nombre} de {self.ciudad}")

    # c) Agrega un método para verificar si es mayor de edad
    def verificar_mayor_edad(self):
        if self.edad >= 18:
            print("Mayor de edad")
        else:
            print("Menor de edad")


# b) Crea tres personas y muestra su saludo
if __name__ == "__main__":
    p1 = Persona("Carol", 18, "La Paz")
    p2 = Persona("Maria", 22, "Cochabamba")
    p3 = Persona("Juan", 15, "Chuquisaca")

    p1.saludo()
    p2.saludo()
    p3.saludo()

    p1.verificar_mayor_edad()
    p2.verificar_mayor_edad()
    p3.verificar_mayor_edad()